utils/rest-swagger/pom.xml

```diff
@@ -105,6 +105,24 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-xml</artifactId>
+            <exclusions>
+                <exclusion>
+                    <!-- jackson above v 2.10.0 wants 6.0 of woodstox, but cxf used elsewhere is only at 5.x -->
+                    <groupId>com.fasterxml.woodstox</groupId>
```

Comment:
We should be really careful about using these kinds of excludes. They only apply to the maven build.

Excluding this tell us that jackson 2.10.0 has been compiled against a different version of woodstox than is shipped in karaf (and in the Brooklyn distro), so we risk `NoSuchMethoError`s etc.

These kind of exclusions are a worry to me: it's safer to just use the same version, if possible.

Also, they only apply to the maven build and not to karaf. Depending on what the feature.xml says, we might end up with both or just the pre-installed version.